collectors/python.d.plugin/anomalies/anomalies.chart.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
+# Description: anomalies netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import time
+from datetime import datetime
+import re
+
+import requests
+import numpy as np
+import pandas as pd
+from netdata_pandas.data import get_data, get_allmetrics
+from pyod.models.hbos import HBOS
+from pyod.models.pca import PCA
+from pyod.models.loda import LODA
+from pyod.models.iforest import IForest
+from pyod.models.cblof import CBLOF
+from pyod.models.feature_bagging import FeatureBagging
+from pyod.models.copod import COPOD
+from sklearn.preprocessing import MinMaxScaler
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+
+priority = 50
+update_every: 2
```

Comment:
I am using `pyton 3.9` and after to configure the `python.d/anomalies.conf` and start Netdata with default values, I am seeing this error:

```
 python.d WARNING: plugin[main] : [anomalies] error on loading source : SyntaxError('invalid syntax', ('/usr/libexec/netdata/python.d/anomalies.chart.py', 27, 13, 'update_every: 2\n')), skipping it
```

I think you wanted to write `update_every = 2`.

Reply:
will take this out of the `.py` file and have it be just in the `.conf` as that's how I see other collectors doing it.